Sources/Fluent/PageLimit.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public struct PageLimit: Codable, ExpressibleByIntegerLiteral {
+    public let value: Int?
+    
+    public init(integerLiteral value: IntegerLiteralType) {
+        self.value = value
+    }
+
+    private init(value: Int?) {
```

Comment:
Don't we need this to be public in order to be able to initialize from a variable?